lib/index.js

```diff
@@ -35,38 +35,27 @@
     return disabled !== false && disabled !== null;
   };
 
-  var focusAndCheck = function (event, target) {
+  var focus = function (event, target) {
     var myTarget = target || event.currentTarget;
 
-    if (isDisabled(myTarget)) {
+    if (!myTarget || isDisabled(myTarget)) {
       return;
     }
 
     myTarget.focus();
-
-    switch (myTarget.type) {
-      case 'checkbox':
-        myTarget.checked = !myTarget.checked;
-        event.preventDefault();
-        break;
-      case 'radio':
-        myTarget.checked = true;
-        event.preventDefault();
```

Comment:
Removing preventDefault allows checkboxes to be checked with a standard event. No need to manually check them.